src/lib/draw/AttributeMode.svelte

```diff
@@ -21,7 +23,44 @@
   }
 
   // Calculate hover
-  $map.on("mousemove", (e) => {
+  $map.on("mousemove", onMouseMove);
```

Comment:
This is an example of the change within a component. Each event handler should be a function, so we can refer to it for `on` and `off`